docs/grammar.txt

```diff
@@ -180,6 +184,7 @@ AssemblyCase = 'case' Literal AssemblyBlock
 AssemblyDefault = 'default' AssemblyBlock
 AssemblyForLoop = 'for' AssemblyBlock AssemblyExpression AssemblyBlock AssemblyBlock
 AssemblyBreakContinue = 'break' | 'continue'
+AssemblyLeave = 'leave'
```

Comment:
Did you add this to the yul grammar too (`docs/yul.rst`)?

Reply:
That's up last in the list: https://github.com/ethereum/solidity/issues/7970 ;)